visualization/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:12.18.2-alpine
+
+COPY . /code
+
+WORKDIR /code
+
+ENV PATH /code/node_modules/.bin:$PATH
```

Comment:
Same comment as above about installing node_modules here, instead of relying on a developers local file system organization

Reply:
Again it just adds it to the node js PATH and installs the node modules in the directory itself. You can have a look here
https://github.com/PecanProject/BETYdb-YABA/blob/676e63deffe05d90fd65c49f4294c4fddf11b183/visualization/Dockerfile#L9